sdk/ai/Azure.AI.Projects/CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 
 ### Features Added
 
+* Added objects to inform Agents about Azure Functions.
```

Comment:
Can we update this to mention that `AzureFunctionToolDefinition` support has been added, similar to `OpenApiTool`?

Reply:
Fixed.